client/components/RelatedProducts/RelatedProducts.jsx

```diff
@@ -1,23 +1,56 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import RelatedProductsList from './Related/RelatedProductsList.jsx';
 import CustomOutfitList from './Outfit/CustomOutfitList.jsx';
 import Grid from '@material-ui/core/Grid';
-import products from './../DummyData/ProductDummyData';
+import axios from 'axios';
+import config from '../../../config.js'
 
+axios.defaults.headers.common['Authorization'] = config.GITHUB_TOKEN;
```

Comment:
Cool, I didn't even know you could set up common headers!

Reply:
Oh yeah, I found that because I had so many get requests to make!  Meant to share with the team.  I'll drop it in the chat.